.github/workflows/deploy-preview.yml

```diff
@@ -2,11 +2,9 @@ name: Preview Deployment
 on: [pull_request]
 
 env:
-  NEON_DATABASE_USERNAME: ${{ secrets.NEON_DATABASE_USERNAME }} # change this to your database username
-  GH_TOKEN: ${{ secrets.GH_TOKEN }} # Required for commenting on pull requests for private repos
-  NEON_API_KEY: ${{ secrets.NEON_API_KEY }} # You can generate a an API key in your account settings
-  NEON_PROJECT_ID: ${{ secrets.NEON_PROJECT_ID }} # You can find this in your project settings
+  NEON_PROJECT_ID: ${{ var.NEON_PROJECT_ID }} # You can find this in your Neon project settings
```

Comment:
This will require you moving the `NEON_PROJECT_ID` from secrets over to vars but it's not a protected secret so I think that's fine.

Reply:
If we move `NEON_PROJECT_ID` to vars it no longer needs to be listed up here. However I think it's good to have here so we can explain where it came from.

Reply:
Yea I'm a fan of listing variables at the top of the workflow